src/client/public/locale/fr/common.json

```
@@ -175,8 +175,9 @@
   "importDate": "Date d'importation",
   "importedBy": "Importé par",
   "importedData": "Données importées",
-  "boreholesImported": "forages ont été importés",
+  "boreholesImported": "Forages ont été importés",
```

REQ: ich denke es war vorher richtig, da das immer in Kombination mit einer Zahl verwendet wird. zb.
6 forages été importés
```suggestion
  "boreholesImported": "forages ont été importés",
```